projects/novo-elements/src/elements/form/extras/address/Address.ts

```diff
@@ -85,6 +85,17 @@ export interface NovoAddressConfig {
 export class NovoAddressElement implements ControlValueAccessor, OnInit {
   @Input()
   config: NovoAddressConfig;
+  private _readOnly: boolean = false;
+  @Input()
+  set readOnly(readOnly: boolean) {
+    this._readOnly = readOnly;
+    this.fieldList.forEach((field: string) => {
+      this.disabled[field] = this.readOnly;
+    });
```

Comment:
this will interfere with some logic we have around disabling state if country isn't selected or if there are no states for a selected country. we need to account for that scenario as well.

Reply:
how so?  If it is readOnly you can't change any values?

Reply:
if you enable it then you can right, and the FI allows you to enable the address picker back

Reply:
You're right, that can leave the state picker enabled when it shouldn't be. Pushing up a fix now.